### ISSUE TITLE
No GPU load in 0.2.1?

### ISSUE THREAD

**Issue by darwiniandude** [divamgupta/diffusionbee-stable-diffusion-ui#114]
I have a 48-core GPU and 2-core CPU in my M1 Ultra Mac Studio. The first built (the only other i've tried) would leave the CPU cores alone when running for the most part but max out the GPU cores. I fired up the new version today and told it to make 4 images, 512x768, 50 steps, 7.5 i think... anyway the GPU isn't doing anything and all the CPU cores are maxed out while running in Activity monitor. Also it's using 4GB ram where it used to used 45GB for a similar (single image) prompt. 
I'm guessing these changes make it behave better on lower spec'd machines but it's a downgrade in speed on mine for sure.

If this is a deliberate change can there be a preference, or can it use both CPU and GPU?

Thanks :)

EDIT: Oh wow, I thought it would be making four files at once, no it's one after the other. It is WAY slower than before then, very considerably so.

**Comment by darwiniandude** [comment]
Just realised that the new version prompt sent me to 0.2.1, will try 0.2.2

**Comment by darwiniandude** [comment]
OK it's fixed in 0.2.2, only downloaded 0.2.1 as that's what happened when I clicked to get the new update. Speedy under 0.2.2. Thanks :) Will close this issue now.

**Comment by remorses** [comment]
Hey, how much time does it take to generate an 1000x1000 image with your machine? 

I am trying to decide what Mac to buy

**Comment by darwiniandude** [comment]
I don't get the option of 1000x1000? I normally do wider images, eg 768x512 then ML upscale in pixelmator pro. However for ease of comparison I've set to 768x768 with version 0.3.0 of diffusion bee, using the prompt on the diffusionbee website "a giant bear with antlers nightmare, art by Gerhard Richter" at 25 steps guidance 7.5 it takes 53 seconds. 32 seconds for 768x512. 22 seconds for 512x512. 512x512 at 75 steps takes 39 seconds. 768x768 at 75 steps takes 2:48. Hopefully this is helpful. This is a base model M1 Ultra mac studio, that is, 64GB ram and 48-core GPU. I did get more storage though.
